includes/cancel-on-next-payment-date.php

```diff
@@ -196,10 +197,7 @@ function pmproarc_conpd_gettext_cancel_text( $translated_text, $text, $domain )
  *
  * @return string The updated email body content.
  */
-function pmproarc_conpd_pmpro_email_body( $body, $email ) {
-	if ( function_exists( 'pmproconpd_pmpro_email_body' ) ) {
-		return $body;
-	}
+function pmproconpd_pmpro_email_body( $body, $email ) {
```

Comment:
Check if this function is already defined/exists

Reply:
Decision made in Slack that this check is not necessary.